product_warranty/models/product_supplierinfo.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# ########################################################################
+#                                                                        #
+#                                                                        #
+# ########################################################################
+#                                                                        #
+# Copyright 2015 Vauxoo
+# Copyright (C) 2009-2011  Akretion, Emmanuel Samyn, Benoît Guillot      #
+#                                                                        #
+# This program is free software: you can redistribute it and/or modify   #
+# it under the terms of the GNU General Public License as published by   #
+# the Free Software Foundation, either version 3 of the License, or      #
+# (at your option) any later version.                                    #
+#                                                                        #
+# This program is distributed in the hope that it will be useful,        #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of         #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the          #
+# GNU General Public License for more details.                           #
+#                                                                        #
+# You should have received a copy of the GNU General Public License      #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.  #
+##########################################################################
+
+from openerp import api, fields, models
+
+
+class ProductSupplierInfo(models.Model):
+
+    _inherit = "product.supplierinfo"
+
+    @api.model
+    def get_warranty_return_partner(self):
+        result = [('company', 'Company'),
+                  ('supplier', 'Supplier'),
+                  ('other', 'Other'),
+                  ]
+        return result
+
+    @api.model
+    def _get_default_instructions(self):
+        """ Get selected lines to add to exchange """
```

Comment:
This docstring doesn't make sense (what selected lines, what exchange?), I see it was already there before the porting, I think this could be a good occasion to change or remove it.